bin/critterget.py

```diff
@@ -554,6 +554,50 @@ def getDailyCrashes(appId,appName):
     return
 
 
+def getTrends(appId,appName):
+    """Get the number of daily app crashes for all versions of an app."""
+
+    trends_data = apicall(u'{}/trends'.format(appId))
+
+    getTopValues(appId,appName,trends_data)
+    getTimeseriesTrends(appId,appName,trends_data)
+
+
+def getTopValues(appId,appName,trendsData):
+    "Get top values from the trends endpoint"
+
+    trend_names = [u'appLoadsByVersion',
+                 u'crashesByVersion',
+                 u'appLoadsByOs',
+                 u'crashesByOs']
+
+    for trend in trend_names:
+        messages = u''
+        for version, value in trendsData[u'series'][trend][u'todayTopValues'].iteritems():
```

Comment:
Just an FYI here, but if you're defining a string without variable content and it doesn't have special chrs in it, no reason to use the 'u'.